mathesar/analytics.py

```diff
@@ -137,3 +141,11 @@ def delete_stale_reports():
             - timezone.timedelta(days=ANALYTICS_REPORT_MAX_AGE)
         )
     ).delete()
+
+
+def upload_initial_report():
```

Comment:
Is this wired somewhere? I couldn't figure out where this is getting called.

Reply:
This would be called from a django template which is yet to be implemented.